blueprints/ember-cli-blanket/files/tests/blanket-options.js

```diff
@@ -0,0 +1,18 @@
+/*globals blanket, module */
+
+var options = {
+  modulePrefix: "<%= dasherizedPackageName %>",
+  filter: "//.*<%= dasherizedPackageName %>/.*/",
+  antifilter: "//.*(tests|template).*/",
+  loaderExclusions: [],
+  enableCoverage: true,
+  cliOptions: {
+    outputFile: 'test-output.json',
```

Comment:
I'm going to create reporter specific options so output files would be under `jsonOptions:{}`